pages/api/player/create.ts

```diff
@@ -2,41 +2,40 @@ import prisma from "../../../lib/prisma";
 import type { NextApiRequest, NextApiResponse } from "next";
 
 type PlayerFormData = {
-    userId: string
-    tournamentId: string
-    alias: string
-    address: string
-    learningInstitution?: string
-    eyeColor?: string
-    hair?: string
-    height?: string
-    other?: string
-    security?: string
-    calendar?: object
-    title?: "LL" | "KK" | "MM" | "TT"
-}
+  userId: string;
+  tournamentId: string;
+  alias: string;
+  address: string;
+  learningInstitution?: string;
+  eyeColor?: string;
+  hair?: string;
+  height?: string;
+  other?: string;
+  security?: string;
+  calendar?: object;
+  title?: "LL" | "KK" | "MM" | "TT";
```

Comment:
Tässäkin voisi periaatteessa viitata noihin constantsin vakioihin string-literaalien sijaan, vaikkei ero tässä yhteydessä suuren suuri olekaan.

Reply:
Ah joo, voi olla että noi pelaajatittelit pitäis ensin määritellä jonain enumina siellä constantsissa niin sit siihen vois viitata täällä tyyppinä.

Reply:
Eli jos muuttaa constantsin `.ts`:ksi ja tekee sinne vaikkapa seuraavanlaisen enum-määrittelyn:
```typescript
export enum PlayerTitle {
  KK = "KK",
  MM = "MM",
  LL = "LL",
  TT = "TT"
}
```
ja importtaa sen sieltä, niin tässä pitäis voida sanoa `title?: PlayerTitle`. Samalla myös PlayerDetailsFormissa voisi viitata tuon enumin arvoihin <options>-luonneissa malliin `<option value={PlayerTitle.KK}>{PlayerTitle.KK}</option>`. Sitten tuolta constantseista voisi pudottaa nuo irrallisena määritellyt arvot ja selvennyskommentin.

Reply:
Nuo enumin avaimet nyt periaatteessa voisi olla jotain selkeämpääkin, mutta se on toisen päivän murhe.